udata_front/theme/gouvfr/assets/js/composables/useDropdown.ts

```diff
@@ -2,7 +2,7 @@ import { ref, onMounted, onUnmounted, MaybeRefOrGetter, toValue } from 'vue';
 import { useCollapse } from './useCollapse';
 import useActiveDescendant from './useActiveDescendant';
 
-interface Option {
+type Option = {
```

Comment:
We shouldn't define the same type twice (here and in `useActiveDescendant`). It should be exported